app/views/local_transactions/_fields.html.erb

```diff
@@ -48,6 +48,43 @@
           <%= f.text_area :need_to_know, rows: 4, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
         </span>
       </div>
+
+      <% administrations = {
+        "Scotland" => :scotland_availability,
+        "Wales" => :wales_availability,
+        "Northern Ireland" => :northern_ireland_availability
+      } %>
+      <div class="devolved-administrations">
+        <% administrations.each do |title, field_name| %>
+          <%= f.fields_for field_name do |f| %>
+            <div class="form-group">
+              <div class="form-label">
+                <%= f.label :type, title %>
```

Comment:
This shouldn't be a label as there is id="input" for it to link to it. I think a header would be the appropriate choice for it. You could have a look for any similar precedents, most semantically I imagine this would be a legend inside a fieldset but that might be a whole can of worms without precedent.

Reply:
Fixed up - is now a govuk heading